domain-server/src/DomainServer.h

```diff
@@ -27,6 +27,8 @@
 #include <Assignment.h>
 #include <HTTPSConnection.h>
 #include <LimitedNodeList.h>
+#include <shared/WebRTC.h>
+#include <webrtc/WebRTCSignalingServer.h>
```

Comment:
these includes could be behind the ifdef

Reply:
The `WebRTCSignalingServer.h` one can be but the `WebRTC` one can't be because that's where the `WEBRTC_DATA_CHANNEL` define is set.